packages/fast-tooling-react/src/viewer/utilities/design-system.ts

```diff
@@ -0,0 +1,24 @@
+import { Direction } from "@microsoft/fast-web-utilities";
+import { memoize } from "lodash-es";
+
+export interface DesignSystem {
+    /**
+     * The primary direction of the view.
+     */
+    direction: Direction;
+}
+
+const designSystemDefaults: DesignSystem = {
+    direction: Direction.ltr,
+};
+
+/**
+ * Ensure that all properties of the design system are assigned
+ */
+export const withDesignSystemDefaults: (
```

Comment:
We've been using `withDefaults` from `fast-jss-utilities` to create this elsewhere. Since we have a dependency should we leverage that helper?